openedx/core/djangoapps/enrollments/views.py

```diff
@@ -50,6 +53,9 @@
 from common.djangoapps.student.models import CourseEnrollment, User
 from common.djangoapps.student.roles import CourseStaffRole, GlobalStaff
 from common.djangoapps.util.disable_rate_limit import can_disable_rate_limit
+from lms.djangoapps.courseware.courses import get_course
+from lms.djangoapps.courseware.models import StudentModule, BaseStudentModuleHistory
```

Comment:
> I know you're in a different cell now, so I'll see if @symbolist or @eLRuLL would be able to provide any information about that.

Nope, I do not have any context about that commit.

Reply:
no idea :grimacing:

Reply:
Alright thanks for the reply though @symbolist @eLRuLL 👍🏼

Reply:
>There's a duplicate import statement.

Thanks for catching this, @nizarmah!

>OpenCraft doesn't use Jenkins... Or is this to prepare for the Koa.2 relase?

We have similar commit cherry-picked in `opencraft-release/juniper.3`. But looks like this commit now in `koa.2`.

Reply:
@0x29a I know you just force-pushed changes, but the duplicate import statements are still there, can you please address that?

> But looks like this commit now in koa.2.

Hmm in order to avoid having the same commit merged twice, would you mind not cherry-picking those changes?
Also, regarding the edX logos change, it was also cherry-picked into [koa.2](https://github.com/edx/edx-platform/compare/open-release/koa.1...open-release/koa.2). 

We can directly create `opencraft-release/koa.2`. Based on that, do you think we should include the edx logos cherry pick with `opencraft-release/koa.1`? Not including it would make merging the [changes](https://github.com/edx/edx-platform/compare/open-release/koa.1...open-release/koa.2) from `open-release/koa.2` easier.